group_vars/all.yml

```diff
@@ -19,25 +19,25 @@ sui:
   validator:
     source_url: https://releases.sui.io
     ports:
-      admin_interface_port: 1337
-      network_address_port: 8080
-      narwhal_primary_interface_port: 8081
-      narwhal_worker_interface_port: 8082
-      consensus_address_port: 8083
-      p2p_port: 8084
-      json_rpc_port: 9000
-      metrics_port: 9184
+      prefix: 80
+      suffix:
+        admin: 37
+        network: 80
+        consensus: 81
+        p2p: 84
+        rpc: 90
+        metrics: 91
   fullnode:
     source_url: https://releases.sui.io
     ports:
-      admin_interface_port: 1337
-      network_address_port: 8080
-      narwhal_primary_interface_port: 8081
-      narwhal_worker_interface_port: 8082
-      consensus_address_port: 8083
-      p2p_port: 8084
-      json_rpc_port: 9000
-      metrics_port: 9184
+      prefix: 80
+      suffix:
+        admin: 37
+        network: 80
+        consensus: 81
```

Comment:
lets also add a comment saying, "its updated by updating the onchain config, node fetches the epoch config and uses the port, changes dynamically during epoch change"